applications/nrf5340_audio/src/bluetooth/le_audio_bis_gateway.c

```diff
@@ -424,7 +438,13 @@ int le_audio_send(uint8_t const *const data, size_t size)
 		}
 
 		net_buf_reserve(buf, BT_ISO_CHAN_SEND_RESERVE);
-		net_buf_add_mem(buf, &data[i * data_size], data_size);
+
+		if (enc_audio.num_ch == 1) {
+			net_buf_add_mem(buf, &enc_audio.data[0], data_size_pr_stream);
+		} else {
+			net_buf_add_mem(buf, &enc_audio.data[i * data_size_pr_stream],
+				data_size_pr_stream);
+		}
```

Comment:
No need to differentiate this
```suggestion
		net_buf_add_mem(buf, &enc_audio.data[i * data_size_pr_stream], data_size_pr_stream);
```

Reply:
Nice catch.